MAINTAINERS.md

```diff
@@ -1,6 +1,129 @@
 # Introduction
 
-This document provides a list of the maintainers for each area of development in the oneDPL project.
+This document defines the roles available in the oneDPL project and provides a list of the
+[current maintainers](Current Maintainers) for each area of development in oneDPL. The document also defines the
```

Comment:
This displays as the string "[current maintainers](Current Maintainers)" in the preview, so I think there is a syntax issue.

Reply:
Fixed.